spec/features/admin/sort_view_in_organisation_list_spec.rb

```diff
@@ -38,5 +43,23 @@
         expect(page.body).to match(/Xylophones.*Apple Cakes.*Silly Hats/m)
       end
     end
+
+    context 'and wants to sort by MoU uploads' do
+      before do
+        Organisation.find_by(name: "Department of Silly Hats").signed_mou_attachment.update(created_at: 3.years.from_now)
+      end
+
+      it 'sorts the organistions from oldest to newest when MoU signed is clicked once' do
+        click_link 'MoU Signed'
+
+        expect(page.body).to match(/Apple Cakes.*Xylophones.*Silly Hats/m)
+      end
+
+      it 'sorts the organistions from newest to oldest when MoU signed is clicked twice' do
+        2.times { click_link 'MoU Signed' }
+
+        expect(page.body).to match(/Silly Hats.*Xylophones.*Apple Cakes/m)
```

Comment:
My only comment would be trying to make it more obvious what these organosations are. For instance you could call them: `No Mou`, `New MoU`, `Old Mou`. Or you could name them as variables and have `no_mou.name`, `new_mou.name` and `old_mou.name`. 

Basically just makes it a bit easier to read and understand the order to other developers.